GOVERNANCE.md

```diff
@@ -25,6 +25,8 @@ The following groups automatically qualify for membership and can request to be
 * OpenJS Foundation CPC Members
 * OpenJS Foundation Project Maintainers
 
+Every team member is asked to document ones involvement in any of the standards organizations that person is regularly involved with.
```

Comment:
```suggestion
Every team member is asked to document one's involvement in any of the standards organizations that person is regularly involved with.
```

Reply:
Does it mean that we are restricting the regular members to be a member of an already existing standards org?

Reply:
It says document involvement, not that involvement is required.